src/default_parameters.jl

```diff
@@ -112,7 +112,7 @@
 
     # INITIAL CONDITIONS
     initial_cond::String="rest"         # "rest" or "ncfile" for restart from file
-    initpath::String=outpath            # folder where to pick the restart files from
+    path::String=outpath            # folder where to pick the restart files from
```

Comment:
Why did you rename this? And use it as the output path?

Reply:
Because the SpeedyWeather functions just called it path, but I might have misunderstood what `path` in SpeedWeather was versus ShallowWaters initpath?

Reply:
I see now that that is what happened, I missed that there was both `outpath` and `path` in Parameters

Reply:
Sorry about that